client/require.js

```diff
@@ -0,0 +1,17 @@
+const moduleStore = {
```

Comment:
This is almost the same with https://github.com/zeit/next.js/blob/master/lib/eval-script.js
Can we reuse if it makes sense ?